.github/workflows/release.yml

```diff
@@ -99,3 +99,26 @@ jobs:
     secrets:
       APP_ID: ${{ secrets.APP_ID }}
       APP_PRIVATE_KEY: ${{ secrets.APP_PRIVATE_KEY }}
+
+  push-nuget:
+    name: Push nuget clients
+    needs: validate
+    runs-on: ubuntu-22.04
+    environment: nuget-release
+    steps:
+      - name: Setup the latest .NET 7 SDK
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 7.0.x
```

Comment:
I just realised this step is missing from the `pack-nuget` job! Could you please add it?

Reply:
sure

Reply:
@jgiannuzzi step added